utils.go

```diff
@@ -0,0 +1,24 @@
+package goiscsi
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"regexp"
+)
+
+func checkIPAddress(ip string) error {
```

Comment:
ValidateIPAddress might be more suitable

Reply:
fixed